CONTRIBUTING.md

```diff
@@ -2,22 +2,25 @@
 
 :+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
 
-The following is a set of guidelines for contributing to Forgerock and its packages, which are hosted in the [Forgerock](https://github.com/forgerock) on GitHub.
+The following is a set of guidelines for contributing to Forgerock and its packages, which are hosted in the [ForgeRock organization](https://github.com/forgerock) on GitHub.
```

Comment:
Being picky, I'd say "contributing to Forgerock and its packages" almost implies that they are contributing to the product itself rather than the open source SDKs. I know most people wanting to contribute would know what they're contributing to, but for accuracy would "contributing to the ForgeRock SDK and its packages" be better?